src/EnergyPlus/DataSizing.cc

```diff
@@ -649,88 +653,301 @@ Real64 calcDesignSpecificationOutdoorAir(EnergyPlusData &state,
                                                      UseMinOASchFlag,
                                                      PerPersonNotSet,
                                                      MaxOAVolFlowFlag,
-                                                     thisDSOA.dsoaSpaceIndexes(dsoaCount));
+                                                     thisDSOA.dsoaSpaceIndexes(dsoaCount),
+                                                     calcIAQMethods);
             }
         }
         return totOAFlowRate;
     }
 }
 
-Real64 OARequirementsData::desFlowPerZoneArea(EnergyPlusData &state, int const zoneNum, int const spaceNum)
+int getDefaultOAReq(EnergyPlusData &state)
```

Comment:
New function to return index to "default" OA requirements (previously these defaults were applied directly in the `VentilationMechanical` data structure.